src/DataSet/ObjectDataSet.php

```diff
@@ -13,106 +13,48 @@
 use Yiisoft\Validator\RulesProviderInterface;
 
 use function array_key_exists;
-use function get_class;
 
 /**
  * This data set makes use of attributes introduced in PHP 8. It simplifies rules configuration process, especially for
- * nested data and relations. Please refer to the guide for examples.
+ * nested data and relations. Please refer to the guide for example.
```

Comment:
?

Reply:
Actually currently there are no examples with `ObjectDataSet` in README at all.

Reply:
done